pkg/reconciler/apis/apiexport/apiexport_reconcile.go

```diff
@@ -97,29 +100,43 @@ func (c *controller) reconcile(ctx context.Context, apiExport *apisv1alpha2.APIE
 		)
 	}
 
-	// TODO(sttts): reactivate this with multi-shard support eventually
-	/*
-		// check if any APIBindings are bound to this APIExport. If so, add a virtualworkspaceURL
-		apiBindings, err := c.getAPIBindingsForAPIExport(clusterName, apiExport.Name)
-		if err != nil {
-			return fmt.Errorf("error checking for APIBindings with APIExport %s|%s: %w", clusterName, apiExport.Name, err)
+	// Ensure the APIExportEndpointSlice exists
+	_, err := c.getAPIExportEndpointSlice(clusterName, apiExport.Name)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			// Create the APIExportEndpointSlice
+			apiExportEndpointSlice := &apisv1alpha1.APIExportEndpointSlice{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: apiExport.Name,
+				},
+				Spec: apisv1alpha1.APIExportEndpointSliceSpec{
+					APIExport: apisv1alpha1.ExportBindingReference{
+						Name: apiExport.Name,
+						Path: clusterPath,
+					},
+				},
+			}
+			if err := c.createAPIExportEndpointSlice(ctx, clusterName.Path(), apiExportEndpointSlice); err != nil {
```

Comment:
wasn't it said that this is the default, but can be disabled?

Reply:
Added the annotation